modules/mods_for_hats/mods_for_hats.module

```diff
@@ -224,8 +224,8 @@ function mods_for_hats_spin_the_wheel_yay(&$text, $key) {
           $result = $query->execute();
           $words = array();
           foreach ($result as $record) {
-            $words[] = $record->word;
-            $words[] = $record->definition;
+            $words[] = preg_replace('/[^\\x0009\\x000A\\x000D\\x0020-\\xD7FF\\xE000-\\xFFFD]/', ' ', addslashes($record->word));
+            $words[] = preg_replace('/[^\\x0009\\x000A\\x000D\\x0020-\\xD7FF\\xE000-\\xFFFD]/', ' ', addslashes($record->definition));
```

Comment:
what's the odds we could htmlspecialchars() out the special characters instead so that they don't lose them when populating our MODS?